scripts/bootstrap

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env bash
+
+#
+# bootstrap
+#
+# Install and configure Opkg for use with Entware and Toltec
+#
+
+bootstrap_wget() {
+    if [ ! -d /home/root/.cache/wget_bin ]
+    then
+        log INFO "Installing a current wget binary with better and current tls support" \
+            "to /home/root/.cache/wget_bin/ for this installation..." \
+            ""
+        # Bootstrap a current version of wget
+        WGET_BINARIES_PATH='http://github.com/LinusCDE/wget-remarkable-pipeline/releases/download/job254'
+        WGET_BINARIES_FILENAME='wget-remarkable-pipeline_job245_wget1.20.3.zip'
+        WGET_BINARIES_SHA256='84185a5934e34e25794d439c78dc9f1590e4df12fbf369236f6a8749bf14d67f'
+
+        # Download and compare to hash
+        wget "$WGET_BINARIES_PATH/$WGET_BINARIES_FILENAME" -O "/home/root/$WGET_BINARIES_FILENAME"
+        if ! echo "$WGET_BINARIES_SHA256  /home/root/$WGET_BINARIES_FILENAME" | sha256sum -c -
+        then
+            echo "FATAL: Invalid hash" >&2
+            exit 1
+        fi
+
+        # Ensure to /home/root/.cache/wget_bin exists and is empty
+        if [ -d /home/root/.cache/wget_bin ]
+        then
+            rm -rf /home/root/.cache/wget_bin/*
+        else
+            mkdir -p /home/root/.cache/wget_bin
+        fi
+
+        # Unzip to /home/root/.cache/wget_bin and remove downloaded file
+        unzip "/home/root/$WGET_BINARIES_FILENAME" -d /home/root/.cache/wget_bin -q
+        rm "/home/root/$WGET_BINARIES_FILENAME"
+
+        cat > /home/root/.cache/wget_bin/wget <<EOF
+#!/bin/sh
+LD_LIBRARY_PATH="/home/root/.cache/wget_bin/dist" /home/root/.cache/wget_bin/dist/wget \$@
+EOF
+
+        chmod +x /home/root/.cache/wget_bin/wget
+    fi
+
+    # Ensure this binary is used
+    if [[ $(command -v wget) != /home/root/.cache/wget_bin/wget ]]; then
+        PATH="/home/root/.cache/wget_bin:$PATH"
+    fi
```

Comment:
Shouldn’t the `/home/root/.cache/wget_bin` directory get cleaned up when the script ends?

Reply:
That is probably a good idea. I didn't do it here, so we may reuse it afterwards if needed.

This commit 6330e6e should address this. It's not perfect but do you find it okay this way (feel free to make changes).